backend/src/main/java/gov/cdc/usds/simplereport/utils/DateTimeUtils.java

```diff
@@ -62,9 +63,9 @@ public class DateTimeUtils {
           Map.entry("HI", hawaiiTimeZoneId),
           Map.entry("HST", hawaiiTimeZoneId),
           Map.entry("HDT", aleutianTimeZoneId),
-          Map.entry("AS", samoaTimeZoneId),
```

Comment:
⚠️ removed "AS" and "ASM" because I don't think they are actually timezones for American Samoa based on [this list](https://en.wikipedia.org/wiki/List_of_time_zone_abbreviations) Let me know if I should be using a different list 🤔

Reply:
OOO thank you for letting me know! I'll take a look!

Reply:
Added "AS" and "ASM" back